lib/ui/compositing/scene.h

```diff
@@ -50,7 +50,11 @@ class Scene : public RefCountedDartWrappable<Scene> {
                  bool checkerboardRasterCacheImages,
                  bool checkerboardOffscreenLayers);
 
-  std::unique_ptr<flutter::LayerTree> layer_tree_;
+  void RasterizeToImageSync(uint32_t width,
+                            uint32_t height,
+                            Dart_Handle raw_image_handle);
+
+  std::shared_ptr<flutter::LayerTree> layer_tree_;
```

Comment:
Consider documenting why this is a shared pointer - and that it's no longer valid after takeLayerTree.

Reply:
Done